shotover-proxy/tests/scripts/cassandra-cluster-shot.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
This sounds like a pretty reasonable addition but I think it should live in `examples/cassandra-cluster` as it will be easier to discover there.

Reply:
Given that we are now keeping cassandra-cluster.sh it makes sense for this to be separate to cassandra-cluster.sh and not run `docker-compose -f docker-compose.yml up`.

As for moving it, its not super important and it can stay where it is.